plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -556,3 +592,25 @@ func init() {
 		return &SQLServer{Servers: []string{defaultServer}}
 	})
 }
+
+type QueryDataCache struct {
+	mx sync.RWMutex
+	m  map[string]string
+}
+
+func (c *QueryDataCache) Get(key string) (string, bool) {
+	c.mx.RLock()
+	defer c.mx.RUnlock()
+	val, ok := c.m[key]
+	return val, ok
```

Comment:
```suggestion
	return c.m[key]
```

Reply:
The proposed code change leads to compile error. The Get method should return 2 values: the value itself and success flag. The suggested change leads to the case when the method returns only one value.

Reply:
Sorry for the noise!